policies/membership/auto-alumni.md

```diff
@@ -0,0 +1,52 @@
+# Keeping membership aligned with reality
+
+This policy outlines how the Rust Project will maintain the membership in rust-lang/team to be kept up to date.
+
+## Goals
+
+Why do we care about keeping membership up to date?
+
+* Improved planning (e.g., budgeting for usage of grants and space at events)
+* Elections and other processes seeking consensus don't stumble on 'maybe present' people
+  * i.e., we avoid ad-hoc determinations of where to set the bar by having a consistent bar
+* Providing a clear, easy off-ramp for people who want to step back
+  * Avoiding the *default* being to continue indefinitely
+
+## Identification
+
+Once a year, the Council will ensure communication is sent out to every
+non-alumni person in rust-lang/team. This communication will, in addition to
+any other questions, inform respondents what teams they are *currently listed*
+as a part of. The survey will invite a response of "Please remove me from
+<blank> teams regardless of any other responses."
```

Comment:
With markdown, a tag will be hidden/invisible. Various other options here:

- `\<blank\>`
- `______`
- `{TEAMS}`

```suggestion
$TEAMS teams regardless of any other responses."
```